From 1981dd3e54f5f7d16e7ce7d31d7c854f71dcea4a Mon Sep 17 00:00:00 2001
Message-Id: <1981dd3e54f5f7d16e7ce7d31d7c854f71dcea4a.1297440281.git.jdenemar@redhat.com>
From: Jiri Denemark <jdenemar@redhat.com>
Date: Fri, 11 Feb 2011 13:02:50 +0100
Subject: [PATCH] cgroup: Add documentation for blkiotune elements.

https://bugzilla.redhat.com/show_bug.cgi?id=632492

Add documentation for blkiotune elements.

Signed-off-by: Gui Jianfeng <guijianfeng@cn.fujitsu.com>
(cherry picked from commit ab2def20ffcd7303d49886262203d6771c3b33b2)

Conflicts:

	docs/formatdomain.html.in -- RHEL-specific context
---
 docs/formatdomain.html.in |   14 ++++++++++++--
 1 files changed, 12 insertions(+), 2 deletions(-)

diff --git a/docs/formatdomain.html.in b/docs/formatdomain.html.in
index 437b705..eea2704 100644
--- a/docs/formatdomain.html.in
+++ b/docs/formatdomain.html.in
@@ -198,6 +198,9 @@
   &lt;memoryBacking&gt;
     &lt;hugepages/&gt;
   &lt;/memoryBacking&gt;
+  &lt;blkiotune&gt;
+    &lt;weight&gt;800&lt;/weight&gt;
+  &lt;/blkiotune&gt;
   &lt;memtune&gt;
     &lt;hard_limit&gt;1048576&lt;/hard_limit&gt;
     &lt;soft_limit&gt;131072&lt;/soft_limit&gt;
@@ -223,10 +226,17 @@
         using hugepages instead of the normal native page size, And the
         optional element <code>nosharepages</code> tells the hypervisor
         that share pages (KSM) should be disabled on guest startup</dd>
+      <dt><code>blkiotune</code></dt>
+      <dd> The optional <code>blkiotune</code> element provides the ability
+        to tune Blkio cgroup tunable parameters for the domain. If this is
+        omitted, it defaults to the OS provided defaults.</dd>
+      <dt><code>weight</code></dt>
+      <dd> The optional <code>weight</code> element is the I/O weight of the
+        guest. The value should be in range [100, 1000].</dd>
       <dt><code>memtune</code></dt>
       <dd> The optional <code>memtune</code> element provides details
-      regarding the memory tuneable parameters for the domain. If this is
-      omitted, it defaults to the OS provided defaults.</dd>
+        regarding the memory tunable parameters for the domain. If this is
+        omitted, it defaults to the OS provided defaults.</dd>
       <dt><code>hard_limit</code></dt>
       <dd> The optional <code>hard_limit</code> element is the maximum memory
 	the guest can use. The units for this value are kilobytes (i.e. blocks
-- 
1.7.4

